src/createServer.js

```diff
@@ -31,7 +31,7 @@ function createServer (options = {}) {
   const optVersion = version === undefined || version === false ? require('./version').defaultVersion : version
 
   const mcData = require('minecraft-data')(optVersion)
-  if (!mcData) throw new Error(`unsupported protocol version: ${optVersion}`)
+  if (!mcData) throw new Error(`Unsupported protocol version: ${optVersion}`)
```

Comment:
undo

Reply:
why?

Reply:
because this is a breaking change with little benefit